src/undate/converters/calendars/hebrew/__init__.py

```diff
@@ -0,0 +1,3 @@
+from undate.converters.calendars.hijri.converter import HijriDateConverter
+
+__all__ = ["HijriDateConverter"]
```

Comment:
We want to be able to import `HijriDateConverter` from `undate.converters.calendars.hebrew`?